board list
             日期     概念名称  ...                                             网址      代码
0    2024-04-03  同花顺出海50  ...  http://q.10jqka.com.cn/gn/detail/code/309127/  309127
1    2024-03-25     AI语料  ...  http://q.10jqka.com.cn/gn/detail/code/309126/  309126
2    2024-03-20   铜缆高速连接  ...  http://q.10jqka.com.cn/gn/detail/code/309125/  309125
3    2024-03-12    高股息精选  ...  http://q.10jqka.com.cn/gn/detail/code/309124/  309124
4    2024-03-04    AI PC  ...  http://q.10jqka.com.cn/gn/detail/code/309121/  309121
..          ...      ...  ...                                            ...     ...
541        None   京津冀一体化  ...  http://q.10jqka.com.cn/gn/detail/code/300061/  300061
569        None       流感  ...  http://q.10jqka.com.cn/gn/detail/code/300038/  300038
597        None     苹果概念  ...  http://q.10jqka.com.cn/gn/detail/code/300309/  300309
598        None    PM2.5  ...  http://q.10jqka.com.cn/gn/detail/code/300134/  300134
633        None      石墨烯  ...  http://q.10jqka.com.cn/gn/detail/code/300337/  300337

[406 rows x 5 columns]
new
       概念名称_cur      概念名称_new
代码                           
301424      NaN         核污染防治
308614      NaN       阿尔茨海默概念
309081      NaN           减肥药
309084      NaN          BC电池
309085      NaN           光刻机
309086      NaN          星闪概念
309089      NaN         新型工业化
309090      NaN          华为昇腾
309092      NaN          智能座舱
309093      NaN          短剧游戏
309094      NaN        长安汽车概念
309104      NaN         多模态AI
309105      NaN        PEEK材料
309106      NaN          小米汽车
309108      NaN         可控核聚变
309113      NaN   飞行汽车(eVTOL)
309115      NaN          低空经济
309118      NaN  Sora概念(文生视频)
309119      NaN         人形机器人
309120      NaN          AI手机
309121      NaN         AI PC
309124      NaN         高股息精选
309125      NaN        铜缆高速连接
309126      NaN          AI语料
309127      NaN       同花顺出海50

remove
       概念名称_cur 概念名称_new
代码                      
301414       口罩      NaN
308296     送转填权      NaN
308501     超级真菌      NaN
308568  创业板重组松绑      NaN
308651  REITs概念      NaN
308698   核准制次新股      NaN
308719     新冠检测      NaN
308820     新冠治疗      NaN
308888     疫情监测      NaN
308935     方舱医院      NaN
309038    熊去氧胆酸      NaN
309044     蒙脱石散      NaN

name change
             概念名称_cur 概念名称_new
代码                            
301223           在线旅游     旅游概念
307816       国家大基金持股   国家大基金持股
308458          半年报预增    一季报预增
309004  先进封装（Chiplet）     先进封装

number change 376
         概念名称_cur  成分股数量_cur  成分股数量_new
代码                                     
300008      新能源汽车      642.0      773.0
300013        大飞机       89.0      104.0
300018       参股保险       86.0       78.0
300019       网络游戏       68.0       72.0
300023       食品安全       46.0       57.0
...           ...        ...        ...
309062  同花顺中特估100      100.0      100.0
309063   MR（混合现实）       28.0       57.0
309065      英伟达概念       28.0       53.0
309066       空间计算       32.0       48.0
309068       算力租赁       43.0       89.0

[376 rows x 3 columns]

Error: get_bdinfo_ori 300378 ('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))
301780 工业互联网 new
          代码    名称
70   002734  利民股份
208  601500  通用股份
218  603180  金牌厨柜
231  603326  我乐家居
236  603611  诺力股份
276  834599  同力股份

301797 智能物流 new
        代码    名称
8  000911  广农糖业

301799 健康中国 removed
         代码     名称
85  600466  *ST蓝光

302010 腾讯概念 new
         代码    名称
76  300232  洲明科技

302035 人工智能 new
          代码    名称
21   000719  中原传媒
586  688529  豪森智能
595  688607  康众医疗

302045 区块链 new
         代码   名称
57  002528  英飞拓

302059 网络直播 new
          代码    名称
143  600857  宁波中百
158  603326  我乐家居
169  603818  曲美家居

302144 恒大概念 removed
        代码     名称
3  000918  *ST嘉凯

Error: get_bdinfo_ori 302150 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /gn/detail/field/199112/order/desc/page/1/ajax/1/code/302150 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921210>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 302154 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/302154 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921350>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 302166 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/302166 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c984a90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 302174 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/302174 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce89d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 302182 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/302182 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf02c10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 302436 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/302436 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf07bd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 302674 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/302674 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf0ebd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 303944 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/303944 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15bd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 304756 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/304756 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf07c90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 305376 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/305376 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf02d10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 305790 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/305790 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8290>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 305794 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/305794 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c984190>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 306380 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/306380 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15d10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 306398 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/306398 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf19d10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 306750 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/306750 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf19910>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 306912 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/306912 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad6190>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 307408 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/307408 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8ad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 307512 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/307512 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf02890>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 307550 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/307550 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c9a5a10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 307816 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/307816 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15a10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 307816 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/307816 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf0eed0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 307822 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/307822 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf0e610>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 307826 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/307826 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15690>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 307904 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/307904 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c9a5190>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 307940 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/307940 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf19ed0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 307954 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/307954 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921190>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308002 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308002 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8ed0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308014 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308014 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c965ed0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308016 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308016 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921b90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308172 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308172 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf197d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308202 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308202 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c9a51d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308294 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308294 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf151d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308296 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308296 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf199d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308300 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308300 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921890>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308366 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308366 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8c10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308384 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308384 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf0e050>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308428 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308428 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982490>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308432 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308432 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982550>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308438 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308438 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf0e1d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308458 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308458 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921790>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308467 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308467 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15350>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308473 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308473 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad6b90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308477 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308477 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2f1550>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308479 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308479 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31e410>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308491 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308491 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921850>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308501 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308501 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8590>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308507 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308507 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf153d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308517 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308517 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad65d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308525 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308525 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982650>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308527 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308527 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c3ee110>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308531 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308531 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31e3d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308535 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308535 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982a90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308537 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308537 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c264090>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308547 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308547 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8710>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308555 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308555 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921910>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308560 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308560 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c6382d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308566 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308566 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c603b50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308568 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308568 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921690>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308572 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308572 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8590>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308582 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308582 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15490>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308588 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308588 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad6090>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308590 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308590 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982710>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308594 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308594 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31efd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308606 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308606 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31eb90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308612 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308612 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c38c950>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308620 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308620 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad6090>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308624 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308624 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15b50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308628 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308628 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8290>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308630 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308630 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921190>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308632 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308632 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2eba50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308634 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308634 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8150>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308636 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308636 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15f50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308638 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308638 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad6e90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308642 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308642 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c3eeb90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308646 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308646 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c9821d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308651 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308651 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2ebf90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308657 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308657 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31e850>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308659 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308659 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982910>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308661 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308661 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c38c290>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308690 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308690 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad6710>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308692 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308692 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15a90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308694 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308694 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c9219d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308696 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308696 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31c8d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308697 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308697 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8590>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308698 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308698 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15a10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308699 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308699 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c38cf90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308700 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308700 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31c650>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308708 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308708 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982c10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308709 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308709 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2eb410>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308710 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308710 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31e050>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308712 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308712 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982f10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308716 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308716 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c3ee390>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308717 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308717 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad6110>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308718 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308718 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf15b50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308719 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308719 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8150>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308725 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308725 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33df90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308728 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308728 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cf157d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308729 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308729 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad6210>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308731 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308731 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad6e50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308732 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308732 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31c790>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308733 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308733 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c9828d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308734 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308734 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31e990>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308740 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308740 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982490>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308742 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308742 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33dfd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308743 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308743 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31cad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308748 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308748 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c9219d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308752 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308752 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2ebb10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308753 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308753 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c603c10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308760 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308760 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c3ee950>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308761 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308761 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8150>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308774 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308774 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921a10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308775 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308775 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31c150>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308791 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308791 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33d810>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308792 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308792 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2eb990>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308794 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308794 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982750>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308801 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308801 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c3eecd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308803 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308803 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982050>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308805 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308805 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2ebad0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308806 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308806 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33de90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308809 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308809 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31ccd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308810 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308810 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c9218d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308814 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308814 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c603310>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308818 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308818 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8f90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308820 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308820 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921650>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308822 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308822 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31ccd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308828 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308828 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33da50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308829 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308829 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c3eecd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308832 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308832 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982dd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308834 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308834 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8b10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308841 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308841 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982550>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308850 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308850 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2eb610>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308854 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308854 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c5d9a10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308855 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308855 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33d050>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308862 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308862 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c9214d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308864 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308864 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c6032d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308869 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308869 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31eb50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308870 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308870 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c603550>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308874 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308874 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31cc10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308876 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308876 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33d650>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308883 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308883 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c5d9950>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308885 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308885 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2eb650>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308887 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308887 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31ead0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308888 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308888 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982e50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308893 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308893 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982050>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308896 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308896 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2ebbd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308901 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308901 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33de10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308915 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308915 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31c750>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308926 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308926 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31eb90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308931 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308931 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921b50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308935 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308935 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8f10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308941 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308941 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31c9d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308956 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308956 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33dbd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308969 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308969 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c5d9150>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308972 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308972 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31ef50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308977 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308977 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2eb790>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308984 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308984 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982dd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308989 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308989 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c9824d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308991 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308991 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2ebfd0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 308992 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/308992 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31ead0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309000 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309000 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c5d9d50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309004 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309004 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33dc90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309007 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309007 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31c150>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309020 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309020 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c603550>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309022 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309022 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921ed0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309023 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309023 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31cb50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309025 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309025 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c38c290>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309030 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309030 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31e390>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309031 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309031 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2eb650>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309034 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309034 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982550>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309035 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309035 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c638b10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309037 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309037 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982810>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309038 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309038 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2ebb10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309040 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309040 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31e7d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309041 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309041 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c5d9550>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309042 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309042 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33d4d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309044 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309044 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31c050>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309045 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309045 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cad6210>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309046 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309046 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c603550>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309049 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309049 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c38c510>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309050 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309050 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31e110>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309051 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309051 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2eb990>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309055 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309055 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982390>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309056 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309056 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c638e10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309058 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309058 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c982510>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309059 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309059 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c2ebb10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309060 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309060 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c33db50>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309061 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309061 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31e510>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309062 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309062 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c6382d0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309063 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309063 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31cc90>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309065 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309065 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c921f10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309066 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309066 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11cce8f10>: Failed to establish a new connection: [Errno 61] Connection refused'))
Error: get_bdinfo_ori 309068 HTTPConnectionPool(host='q.10jqka.com.cn', port=80): Max retries exceeded with url: /thshy/detail/field/199112/order/desc/page/1/ajax/1/code/309068 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11c31e310>: Failed to establish a new connection: [Errno 61] Connection refused'))
update time 2024-04-21
<module> rt_cost 7995.20 s
